telegram/ext/_applicationbuilder.py

```diff
@@ -1016,6 +1019,42 @@ async def post_shutdown(application: Application) -> None:
         self._post_shutdown = post_shutdown
         return self
 
+    def post_stop(
+        self: BuilderType, post_stop: Callable[[Application], Coroutine[Any, Any, None]]
+    ) -> BuilderType:
+        """
+        Sets a callback to be executed by :meth:`Application.run_polling` and
+        :meth:`Application.run_webhook` *after* executing :meth:`Updater.stop`
+        and :meth:`Application.stop`.
+
+        Tip:
+            This can be used for custom stop logic that requires to await coroutines, e.g.
+            sending message to a chat before shutting down the bot
+
```

Comment:
a seealso linking to post_shutdown and post_init is now starting to be more suitable